e2e/atlas/metrics_processes_test.go

```diff
@@ -0,0 +1,165 @@
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	atlasEntity := "atlas"
+	metricsEntity := "metrics"
+	clusterName := "myReplicaSet"
+
+	err = deployCluster(cliPath, atlasEntity, clusterName)
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	hostname, err := getHostnameAndPort(cliPath, atlasEntity)
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	t.Run("processes", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			atlasEntity,
+			metricsEntity,
+			"processes",
+			hostname,
+			"--granularity=PT30M",
+			"--period=P1DT12H")
+
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+
+		metrics := &mongodbatlas.ProcessMeasurements{}
+		err = json.Unmarshal(resp, &metrics)
+
+		if err != nil {
+			t.Fatalf("unexpected error: %v", err)
+		}
+
+		if metrics.Measurements == nil {
+			t.Errorf("there are no measurements")
+		}
+
+		if len(metrics.Measurements) == 0 {
+			t.Errorf("got=%#v\nwant=%#v\n", 0, "len(metrics.Measurements) > 0")
+		}
+
+	})
+
+	deleteCluster(cliPath, atlasEntity, clusterName)
+
+}
+
+func getHostnameAndPort(cliPath, atlasEntity string) (string, error) {
+	cmd := exec.Command(cliPath,
+		atlasEntity,
+		"processes",
+		"list")
+
+	cmd.Env = os.Environ()
+	resp, err := cmd.CombinedOutput()
+
+	var processes []*mongodbatlas.Process
+	err = json.Unmarshal(resp, &processes)
+
+	if err != nil {
+		return "", fmt.Errorf("unexpected error: %v", err)
+	}
+
+	if len(processes) == 0 {
+		return "", fmt.Errorf("got=%#v\nwant=%#v\n", 0, "len(processes) > 0")
+	}
+
+	return processes[0].Hostname + ":" + strconv.Itoa(processes[0].Port), nil
+
+}
+
+func deployCluster(cliPath, atlasEntity, clusterName string) error {
+	cmd := exec.Command(cliPath,
+		atlasEntity,
+		"clusters",
+		"create",
+		clusterName,
+		"--region=US_EAST_1",
+		"--members=3",
+		"--tier=M10",
+		"--provider=AWS",
+		"--mdbVersion=4.0",
+		"--diskSizeGB=10")
+	cmd.Env = os.Environ()
+	resp, err := cmd.CombinedOutput()
+
+	if err != nil {
+		return fmt.Errorf("unexpected error: %v, resp: %v", err, string(resp))
+	}
+
+	cmd = exec.Command(cliPath,
+		"atlas",
+		"clusters",
+		"watch",
+		clusterName)
+	cmd.Env = os.Environ()
+	resp, err = cmd.CombinedOutput()
+
+	if err != nil {
+		return fmt.Errorf("unexpected error: %v, resp: %v", err, string(resp))
+	}
+
+	if !strings.Contains(string(resp), "Cluster available at:") {
+		return fmt.Errorf("got=%#v\nwant=%#v\n", string(resp), "Cluster available at:")
+	}
+
+	return nil
+}
+
+func deleteCluster(cliPath, atlasEntity, clusterName string) error {
+	cmd := exec.Command(cliPath, atlasEntity, "clusters", "delete", clusterName, "--force")
+	cmd.Env = os.Environ()
+	resp, err := cmd.CombinedOutput()
+
+	if err != nil {
+		return fmt.Errorf("unexpected error: %v, resp: %v", err, string(resp))
+	}
+
+	expected := fmt.Sprintf("Cluster '%s' deleted\n", clusterName)
+	if string(resp) != expected {
+		return fmt.Errorf("got=%#v\nwant=%#v\n", string(resp), expected)
+	}
+	return nil
```

Comment:
similar as before, you already have a test for cluster, you should not be testing it again here, if it returns an error, return the error but avoid double testing things